gbpservice/neutron/services/grouppolicy/drivers/oneconvergence_resource_mapping.py

```diff
@@ -0,0 +1,71 @@
+# One Convergence, Inc. CONFIDENTIAL
+# Copyright (c) 2012-2015, One Convergence, Inc., USA
+# All Rights Reserved.
+#
+# All information contained herein is, and remains the property of
+# One Convergence, Inc. and its suppliers, if any. The intellectual and
+# technical concepts contained herein are proprietary to One Convergence,
+# Inc. and its suppliers.
+#
+# Dissemination of this information or reproduction of this material is
+# strictly forbidden unless prior written permission is obtained from
+# One Convergence, Inc., USA
+
+
+import netaddr
+
+from neutron.common import log
+from neutron.openstack.common import jsonutils
+from neutron.openstack.common import log as logging
+from neutron.plugins.common import constants as pconst
+from neutron.common import exceptions as n_exc
+from neutron.extensions import securitygroup as ext_sg
+from oslo.config import cfg
+
+from gbpservice.neutron.services.grouppolicy.common import constants as gconst
+from gbpservice.neutron.services.grouppolicy.common import exceptions as exc
+from gbpservice.neutron.extensions import group_policy as gp_ext
+
+from gbpservice.neutron.services.grouppolicy.drivers import resource_mapping
+
+LOG = logging.getLogger(__name__)
+
+
+class OneConvergenceResourceMappingDriver(resource_mapping.ResourceMappingDriver):
+    """One Convergence Resource Mapping driver for Group Policy plugin.
+
+    This driver inherits default group policy RMD.
+    """
+
+    def create_external_policy_precommit(self, context):
+        self._reject_shared(context.current, 'external_policy')
+        # REVISIT(ivar): For security reasons, only one ES allowed per EP.
+        # see bug #1398156
+        if len(context.current['external_segments']) > 1:
+            raise exc.MultipleESPerEPNotSupported()
+
+
+    @log.log
+    def delete_policy_target_group_postcommit(self, context):
+        try:
+            self._cleanup_network_service_policy(context,
+                                                 context.current,
+                                                 context.nsp_cleanup_ipaddress,
+                                                 context.nsp_cleanup_fips)
+        except Exception as e:
+            LOG.error(e)
```

Comment:
The internationalization of logging is required for openstack components. Please use Log.error(_()).
To just log exceptions, the following code would do:
except Exception:
    LOG.exception(_("Any message"))